.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+**Description**:
+- ...
+
+**Checklist**:
+- [ ] My PR is ready for code review
+- [ ] I have added some tests, if applicable
```

Comment:
I would add, '"and run the whole test suite"

Reply:
I would also add "even linting tests, this way we can get rid of the next point"